python-service/src/report_service.py

```
@@ -0,0 +1,40 @@
+
+REPORT_ROOT = "/var/lib/reviewer/reports"
+DB_PATH = os.environ.get("REPORT_DB", "reports.db")
+
+
+class ReportService:
+    def __init__(self):
+        self.db = sqlite3.connect(DB_PATH, check_same_thread=False)
+        self._cache = {}
+
+    def search_reports(self, term):
+        query = "SELECT id, title, author FROM reports WHERE title LIKE '%" + term + "%'"
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)